### PR TITLE
Camera frame cropping / scaling

### PR DESCRIPTION
**Pull request by hiroshihorie**
Initially I thought `adaptOutputFormat` would work better but it's not working as expected so I'm processing each frame.

### CODE REVIEW

File: Sources/LiveKit/Extensions/CustomStringConvertible.swift [hunk 1]
```diff
@@ -166,3 +167,14 @@ extension LKRTCRtpEncodingParameters {
             ")"
     }
 }
+
+extension AVCaptureDevice.Format {
+    func toDebugString() -> String {
```
Comment (bcherry) [inline Sources/LiveKit/Extensions/CustomStringConvertible.swift:172]: maybe use CustomDebugStringConvertible and debugDescription?

**Review by bcherry — APPROVED** [review]
Are there any updates to docs / readme necessary to use this improvement?

**Comment by davidliu** [comment]
what was the issue with `adaptOutputFormat`? Would've figured that would do exactly as needed.

**Comment by hiroshihorie** [comment]
@davidliu Most of the time it doesn't respect the dimensions passed in the parameters (sometimes smaller, sometimes larger). Does it adapt exactly to what dimensions you provide in Android ?

**Pull request merged**